extensions/arbitrary-stack-pointer/README.md

```diff
@@ -3,7 +3,7 @@
 **Extension State: Under Development**  
 **Enabled by Default: Yes**  
 **Requires: Base**  
-**CPUID Bit: 8**
+**CPUID1 Bit: 7**
```

Comment:
I think instead of this we could consider CPUID as a large struct spanning more than one register (this is typical for such operations in real ISAs) so the notation for this would be

> **CPUID Bit: CP1.7**

Or something to that effect. What do you think?

Reply:
That's always an option. That's what I was going for with how I specified dependencies in the extensions table.

Reply:
Should be resolved now.